src/docs/forms/components/CalendarInput/CalendarInputDescription.js

```diff
@@ -4,7 +4,7 @@ export default class CalendarInputDescription extends React.Component {
     render() {
         return (
             <p>
-                CalendarInput test description.
+               Calendar Inputs are mainly used for selecting/picking date and time.
```

Comment:
R: "CalendarInput is a simple datepicker component that returns a moment instance value upon selection."